vllm_ascend/worker/model_runner_v1.py

```diff
@@ -321,13 +325,40 @@ def __init__(self, vllm_config: VllmConfig, device: torch.device):
         self.sampler = Sampler()
         self.enable_torchair_graph_mode = False
         self.use_cached_npu_graph = False
+        self.torchair_graph_batch_sizes = []
         additional_config = vllm_config.additional_config
         if additional_config:
             self.enable_torchair_graph_mode = additional_config.get(
                 "enable_graph_mode",
                 False) and self.vllm_config.model_config.use_mla
             self.use_cached_npu_graph = additional_config.get(
                 "use_cached_npu_graph", False)
+            if additional_config.get("trace_recompiles", False):
+                torch._logging.set_logs(recompiles=True)
+            self.torchair_graph_batch_sizes = additional_config.get(
+                "torchair_graph_batch_sizes", [])
+            if not isinstance(self.torchair_graph_batch_sizes, list):
```

Comment:
we have `enable_graph_mode` to controller torchair, here named `torchair_graph_batch_sizes`, w'd better to use the same prefix. How about use `torchair_graph` for all case? @yiz-liu